mod/dfrn_poll.php

```diff
@@ -61,16 +63,15 @@ function dfrn_poll_init(App $a) {
 
 		logger('dfrn_poll: public feed request from ' . $_SERVER['REMOTE_ADDR'] . ' for ' . $user);
 		header("Content-type: application/atom+xml");
-		echo DFRN::feed('', $user,$last_update, 0, $hidewall);
+		echo DFRN::feed('', $user, $last_update, 0, $hidewall);
 		killme();
 	}
 
-	if(($type === 'profile') && (! strlen($sec))) {
-
+	if (($type === 'profile') && (!strlen($sec))) {
```

Comment:
You _could_ have reduced that to ```if (($type === 'profile') && !strlen($sec)) {```

Reply:
Why not `if ($type === 'profile' && !strlen($sec)) {` directly?

Reply:
It's like the difference between ```3 + 4 * 5``` and ```3 + (4 * 5)``` - the result is the same, but the brackets are helpful for the understanding.